packages/ngenuity/README.md

```diff
@@ -0,0 +1,75 @@
+<p align="center">
+  <img src="https://github.com/daltonmenezes/assets/blob/master/images/aura-theme/new-heading.png?raw=true" alt="Aura Theme" width="70%" />
+</p>
+
+<p align="center">
+✨ A beautiful dark theme for ngenuity and another apps
+  <br><br>
+
+  <!-- Patreon -->
+  <a href="https://www.patreon.com/daltonmenezes">
+    <img alt="patreon url" src="https://img.shields.io/badge/support%20on-patreon-1C1E26?style=for-the-badge&labelColor=1C1E26&color=61ffca">
+  </a>
+
+  <!-- version -->
+  <a href="#">
+    <img alt="version" src="https://img.shields.io/badge/version%20-v1.0.0-1C1E26?style=for-the-badge&labelColor=1C1E26&color=61ffca">
+  </a>
+</p>
+
+
+
+# Installation
+
+1. Download one of the available profiles and extract it
+2. Click on `Presets` in the upper right corner
+3. Click on `Add` and select de extracted file
+
+<br/>
+Done! ✨ 🎉
+<br/>
+<br/>
+
+# Profiles
+
+### [João Pedro Alves](https://github.com/joaopealves)
+
+![img]()
```

Comment:
Hey men, I sent you in your private discord a photo that shows some of the my items, do you think the image is ok?